### ISSUE TITLE
Alt text for images

### ISSUE THREAD

**Issue by antiantivirus** [toolkitties/toolkitty#111]
User should be able to submit and update alt text alongside an image for accessibility.

As this will require a change on the protocol level we will not implement this for MVP.
